7_react/react1/week3/homework/todo-list/src/App.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useEffect, useState } from "react";
+import TodoList from "./components/todolist";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function App() {
+  const [timer, setTimer] = useState(0);
+  const [todos, setTodos] = useState([]);
+  const [todoDescription, setTodoDescription] = useState("");
+  const [deadline, setDeadline] = useState(null);
+
+  useEffect(() => {
+    fetch(
```

Comment:
although not an issue it's not a good practice to have an anonymous function inside of the `useEffect` hook 
you can assign the fetch logic to a variable (e.g fetchAndSetTodos) and call it afterward inside of the hooks :)